samples/snippets/writes/write_increment_async.py

```diff
@@ -0,0 +1,30 @@
+# [START bigtable_write_increment_async]
+from google.cloud.bigtable.data import BigtableDataClientAsync
+from google.cloud.bigtable.data.mutations import AddToCell, RowMutationEntry
+from google.cloud.bigtable.data.exceptions import MutationsExceptionGroup
+
+async def write_increment_async(project_id, instance_id, table_id):
+    """Increments a value in a Bigtable table using the async client."""
+    async with BigtableDataClientAsync(project=project_id) as client:
+        table = client.get_table(instance_id, table_id)
+        row_key = "unique_device_ids_1"
```

Comment:
nit: you could avoid encoding this string if you create it as a byte string: `row_key = b"unique_device_ids_1"`

But also, RowMutationEntry accepts strings, and [does conversion internally](https://github.com/googleapis/python-bigtable/blob/72dfdc440c22db0f4c372e6f11a9f7dc83fed350/google/cloud/bigtable/data/mutations.py#L375) as well

Up to you how you want to demonstrate this to users